src/sql/src/func.rs

```diff
@@ -2910,6 +2910,27 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         },
         "decode" => Scalar {
             params!(String, String) => BinaryFunc::Decode => Bytes, 1947;
+        },
+        "parse_ident" => Table {
+            params!(String) => Operation::unary(|_ecx, ident| {
+                Ok(TableFuncPlan {
+                    expr: HirRelationExpr::CallTable {
+                        func: TableFunc::ParseIdent,
+                        exprs: vec![ident, HirScalarExpr::literal_true()],
+                    },
+                    column_names: vec!["parse_ident".into()],
+                })
+            }) => ReturnType::set_of(ScalarType::Array(Box::new(ScalarType::String)).into()),
+                oid::FUNC_PARSE_IDENT_DEFAULT_STRICT;
+            params!(String, Bool) =>Operation::binary(move |_ecx, ident, strict| {
+                Ok(TableFuncPlan {
+                    expr: HirRelationExpr::CallTable {
+                        func: TableFunc::ParseIdent,
+                        exprs: vec![ident, strict],
+                    },
+                    column_names: vec!["parse_ident".into()],
+                })
+            }) => ReturnType::set_of(ScalarType::Array(Box::new(ScalarType::String)).into()), 1268;
```

Comment:
Oh, yeah--I knew this was going to be wrong! Should have dug in a little more--sorry about that. It turns out that PG supports making any scalar function into a table function, which we don't support. Let me see how much work that is.

Reply:
Sadly scalar functions in `FROM` clauses turns out to be a real hassle because of our AST transformations. Just made the offending a test a subquery for the time being.